core/src/main/java/org/pentaho/platform/util/ActionUtil.java

```diff
@@ -91,6 +91,27 @@
     KEY_MAP.put( ActionUtil.QUARTZ_RESTART_FLAG, ActionUtil.INVOKER_RESTART_FLAG );
   }
 
+  public static final String WORK_ITEM_XMS = "JVM_XMS";
+  public static final String WORK_ITEM_XMX = "JVM_XMX";
+
+  /**
+   * Validates a JVM_XMS or JVM_XMX value to make sure it has the correct format.
+   * In case the format is not correct, it uses the default value.
+   *
+   */
+  public static boolean validateJvmXm( String jvmXmVar ) {
+    try {
+      int intValue = Integer.parseInt( jvmXmVar );
```

Comment:
Do we need to check for upper limit of this settings?

Reply:
I will add that check